sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2024_10_01_preview/models/_models.py

```diff
@@ -28025,8 +28025,6 @@ def __init__(
 class ServiceTagDestination(msrest.serialization.Model):
     """Service Tag destination for a Service Tag Outbound Rule for the managed network of a machine learning workspace.
 
-    Variables are only populated by the server, and will be ignored when sending a request.
```

Comment:
discussed offline, avoid update _restclient directly. update the swagger in this SDK repo to match exactly with swagger repo and run regenerate rest client off of that. should be small break-fix matching https://github.com/Azure/azure-rest-api-specs/pull/31070

Reply:
will remove this once Jing's PR is checked in